merqube_client_lib/api_client/base.py

```diff
@@ -89,28 +112,61 @@ def get_index_defs(
         res = self.session.get_collection(url)
         return {i["id"]: i for i in res}
 
-    def get_index_manifest(self, index_name: str) -> Manifest:
+    def create_index(self, index_def: IndexDefinitionPost) -> HTTPResJson:
```

Comment:
i just reorganized these methods to put all the name_or_id ones last, no changes